tokio/src/time/driver/tests/mod.rs

```diff
@@ -0,0 +1,2 @@
+#[cfg(loom)]
+mod test_inner;
```

Comment:
I'm not totally sure why we have a separate inner module, instead of just making the `mod tests` in `time/driver/mod.rs` have `#[cfg(all(test, loom)]` on it. I think the intention behind having separate modules elsewhere is so that we can have a separate module for loom tests and non-loom tests, but here, all the tests use `loom`, so I'm not convinced this is strictly necessary.

Reply:
I buy that. I'll collapse this.

Reply:
Resolved with 21ef90a.